deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/JDKImperativeDeckOfCardsAsListTest.java

```diff
@@ -55,28 +56,32 @@ public void cardsAreImmutable()
     @Test
     public void diamonds()
     {
-        Assertions.assertEquals(13, this.jdkDeck.diamonds().size());
+        Assertions.assertEquals("|A♦|,|2♦|,|3♦|,|4♦|,|5♦|,|6♦|,|7♦|,|8♦|,|9♦|,|10♦|,|J♦|,|Q♦|,|K♦|",
+                Iterate.makeString(this.jdkDeck.diamonds(),","));
```

Comment:
While this is certainly more convenient to write and read, it would be good to be consistent with the code you have for building the comma separated strings in `countBySuit` using `Stream` and `Collectors.joining()`.